psqlgraph/node.py

```diff
@@ -142,6 +142,26 @@ def copy(self):
         )
         return node
 
+    def to_json(self):
+        return {
+            'node_id': self.node_id,
+            'label': self.label,
+            'acl': self.acl,
+            'properties': self.properties,
+            'system_annotations': self.system_annotations,
+            'edges_out': [e.to_json() for e in self.edges_out],
+            'edges_in': [e.to_json() for e in self.edges_in]
+        }
+
+    @classmethod
+    def from_json(cls, node_json):
+        Type = Node.get_subclass(node_json['label'])
```

Comment:
Might be good to check that a type was found here; `NoneType object is not callable` might be confusing to debug